packages/template-retail-react-app/package.json

```diff
@@ -13,7 +13,7 @@
     "build-translations": "npm run extract-default-translations && npm run compile-translations",
     "compile-translations": "formatjs compile-folder --ast app/translations app/translations/compiled",
     "compile-translations:pseudo": "formatjs compile --ast app/translations/en-US.json --out-file app/translations/compiled/en-XB.json --pseudo-locale en-XB",
-    "extract-default-translations": "formatjs extract 'app/**/*.{js,jsx}' --out-file app/translations/en-US.json --id-interpolation-pattern [sha512:contenthash:base64:6]",
+    "extract-default-translations": "node ./scripts/build-translations.js",
```

Comment:
How about renaming the script file to "extract-default-translations.js" instead? Because in the context of these npm scripts, "build-translations" is for another npm script.

Reply:
agreed, I've changed the name as suggested